### PR TITLE
fix: adjust ci to external contribution

### PR DESCRIPTION
**Pull request by uziasferreirazup**
## Description

Skipping the `Danger` and `Codecov` when not found the variable this mean someone external contribution open the PR.

## Related Issues
fix issue: #256 

## Tests

Don't has test to this part, I just run the operations local and remote to check if the behavior continues works

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I signed the [DCO].
- [ ] All existing and new tests are passing.
- [X] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [X] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*

[issue database]: https://github.com/ZupIT/beagle/issues
[DCO]: https://developercertificate.org/

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: fastlane/Fastfile [hunk 1]
```diff
@@ -156,8 +128,50 @@ platform :backend do
...
+  "\nRelease notes #{ ENV["VERSION_DEPLOY"] }
+   \nChanges:
+   \n#{changelog}"
+end
+
+private_lane :run_codecov do |options|
+  codecov = ENV["CODECOV_TOKEN"]
+  if !(codecov.nil? || codecov.empty?)
+     commit_hash = last_git_commit[:commit_hash]
+     codecov_reporter(token: ENV["CODECOV_TOKEN"], flags: "-F #{options[:platform]} -C #{commit_hash}")
```
Comment (LG95) [inline fastlane/Fastfile:154]: Can't you use the variable declared above?
```suggestion
     codecov_reporter(token:codecov, flags: "-F #{options[:platform]} -C #{commit_hash}")
```

File: android/beagle/src/main/java/br/com/zup/beagle/android/components/Button.kt [hunk 2]
```diff
@@ -37,6 +37,7 @@ data class Button(
     val text: Bind<String>,
     val styleId: String? = null,
     val onPress: List<Action>? = null,
+
```
Comment (LG95) [inline android/beagle/src/main/java/br/com/zup/beagle/android/components/Button.kt:40]: What? Why?

Comment (uziasferreirazup) [reply android/beagle/src/main/java/br/com/zup/beagle/android/components/Button.kt:40]: Check the description of PR I am testing

Comment (uziasferreirazup) [reply android/beagle/src/main/java/br/com/zup/beagle/android/components/Button.kt:40]: To run CI I need created the PR, draft not work.

Comment (LG95) [reply android/beagle/src/main/java/br/com/zup/beagle/android/components/Button.kt]: I only noticed the WIP after my review, which was kinda early. Sorry.

**Pull request merged**